logger_tt/core.py

```diff
@@ -125,6 +125,7 @@ def _replace_with_queue_handler(self):
             # add queue handler
             queue = self.qclass()
             q_handler = handlers.QueueHandler(queue)
+            q_handler.setFormatter(all_handlers[0].formatter)
```

Comment:
Here, we assume that all handlers use the same formatter.
It may not be always correct but let's just go with it for now.
I'll make a note in the README later.